app/views/logbook_entries/_form.html.erb

```diff
@@ -10,124 +10,67 @@
     </div>
   <% end %>
 
-  <div class="mb-3">
-    <%= form.label :date, class: "form-label" %>
-    <%= form.datetime_field :date, class: "form-control", required: true %>
-    <div class="invalid-feedback">
-      Please provide a valid date.
-    </div>
+  <%= form.datetime_field :date, label: "Date", required: true %>
+  <div class="invalid-feedback">
```

Comment:
Not sure this is needed?

Reply:
Nope it is not needed, looks like I forgot to remove that